lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -24,9 +24,27 @@ def fragment_cache(cached_hash, non_cached_hash)
         private
 
         def serializable_hash_for_collection(options)
+          options.merge!(batch_cache(options))
+
           serializer.map { |s| Attributes.new(s, instance_options).serializable_hash(options) }
         end
 
+        def batch_cache(options)
+          return {} if options[:batch_cache].present? || ActiveModelSerializers.config.cache_store.blank?
+
+          keys = CachedSerializer.object_cache_keys(serializer, @include_tree)
+
+          return {} unless keys.present?
+
+          { batch_cache: ActiveModelSerializers.config.cache_store.read_multi(*keys) }
+        end
+
+        def batch_cached_serializer(options)
```

Comment:
I know I wrote this, but I was just guessing at names without knowing the return type.  Would you mind adding a comment what it returns and maybe change the name of the method?

e.g. using YARDoc

```ruby
# @return [Hash] cached attributes
def batch_cached_serializer(options)
```